config/csi-rbac/rbd_ctrlplugin_role.yaml

```diff
@@ -18,3 +18,6 @@ rules:
   - apiGroups: ["apps"]
     resources: ["deployments/finalizers", "daemonsets/finalizers"]
     verbs: ["update"]
+  - apiGroups: ["authentication.k8s.io"]
+    resources: ["tokenreviews"]
+    verbs: ["create"]
```

Comment:
GitHub is complaining that no new line at the end of the file, can you please check this